tests/test_utils.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import torch
+import torch.nn as nn
+import gym
+import os
+from shutil import rmtree
+
+from jigglypuffRL import (
+    MlpActorCritic,
+    MlpPolicy,
+    MlpValue,
+    PPO1,
+)
+from jigglypuffRL.common.utils import *
+
+
+class TestUtils:
+    def test_get_model(self):
+        """
+        test getting policy, value and AC models
+        """
+        ac = get_model("ac", "mlp")
+        p = get_model("p", "mlp")
+        v = get_model("v", "mlp")
```

Comment:
We should also check the functions `get_value_from_name_`, etc before testing `get_model`. (Since `get_model` is dependent on it.)

Reply:
I know these are not part of `utils.py` but its sort of a shortcoming of the structure, so we have to test them to test `get_model`

Reply:
Won't it get resolved once we add tests for `get_x_from_name` within `policies.py`, `values.py` and `actor_critic.py` since they will fail if those methods don't work?

Reply:
I think I didn't understand